e2e/api_test.go

```diff
@@ -85,6 +85,8 @@ func TestAPI(t *testing.T) {
 	require.NoError(t, err)
 	httpclient, err := rpcclient.NewHTTP("http://localhost:26657", "/websocket")
 	require.NoError(t, err)
+	require.NoError(t, httpclient.OnStart())
```

Comment:
We should not call `OnStart`. It will be called by `Start`. The tendermint node is using the same `service` struct. From outside, we should only call `Start` and `Stop`
```suggestion
	require.NoError(t, httpclient.Start())
```